docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -514,61 +519,69 @@ Time between gossiping accepted frontiers. Defaults to `10s`.
 
 Timeout before killing an unresponsive chain. Defaults to `5s`.
 
-#### `--creation-tx-fee` (int):
+#### `--create-asset-tx-fee` (int):
+
+Transaction fee, in nAVAX, for transactions that create new assets. This can only be changed on a local network.
```

Comment:
```suggestion
Transaction fee, in nAVAX, for transactions that create new assets. Defaults to `10000000` nAVAX (.01 AVAX) per transaction. This can only be changed on a local network.
```